zerver/webhooks/trello/view/card_actions.py

```diff
@@ -64,20 +64,23 @@ def get_proper_action(payload, action_type):
     # type: (Mapping[str, Any], Text) -> Text
     if action_type == 'updateCard':
         data = get_action_data(payload)
+        old_data = data['old']
+        card_data = data['card']
         if data.get('listBefore'):
             return CHANGE_LIST
-        if data.get('old').get('name'):
+        if old_data.get('name'):
             return CHANGE_NAME
-        if data.get('old').get('due', False) is None:
+        if not (old_data.get('due')):
```

Comment:
I don't think its equal and I'm afraid that might break something -  right now code for:
`old_data = {'due': None}` - returns `True` (old one returned `True`)
`old_data = {'not_due': None}` - returns `True` (old one returned `False`)

Also you don't need partenthesis here